api/services/userService.js

```diff
@@ -0,0 +1,37 @@
+const { userDao } = require('../models')
+const jwt = require('jsonwebtoken')
+
+const { getAccessTokenByKakao, getUserInfoByKakao } = require('./kakaoService')
+
+const kakaoSignIn = async(code) => {
+
+    const kakaoAccessToken = await getAccessTokenByKakao(code)
+    const kakaoUserInfo = await getUserInfoByKakao(kakaoAccessToken)
+
+    const socialId = kakaoUserInfo.sub
+    const userInfo = await userDao.getUserSocialId(socialId)
```

Comment:
```suggestion
    const user = await userDao.getUserBySocialId(socialId);
```
변수명, 함수명 수정하면 좋을 것 같아요!